### ISSUE TITLE
Exporting documentation

### ISSUE THREAD

**Issue by GPMueller** [spirit-code/spirit#613]
<div type='discussions-op-text'>

<sup>Originally posted by **Sid281216** February  8, 2022 in discussion #612</sup>
Hi, 

Equations in the documentation does not appear : in .mk files after build on windows, and on the readthedocs website https://spirit-docs.readthedocs.io/en/latest/core/docs/Definitions.html . But it works from GitHub repo website, there is a way to download a pdf of the docs directly from GH and still have equations ?</div>

**Comment by GPMueller** [comment]
The svg document contains
```
  * {
    fill: black;
  }
  @media (prefers-color-scheme: dark) {
    * {
      fill: white;
    }
  }
```
You can check this for example by opening [this equation](https://math.vercel.app/?from=%5Cbegin%7Balignedat%7D%7B2%7D%0A%20%20%20%20%5Cdfrac%7B%5Cpartial%20%5Cvec%7Bn%7D_i%7D%7B%5Cpartial%20t%7D%0A%20%20%20%20%20%20%20%20%3D%26%20-%20%5Cdfrac%7B%5Cgamma%7D%7B%281%2B%5Calpha%5E2%29%5Cmu_i%7D%20%5Cvec%7Bn%7D_i%20%5Ctimes%20%5Cvec%7BB%7D%5E%5Cmathrm%7Beff%7D_i%20%0A%20%20%20%20%20%20%20%20-%20%5Cdfrac%7B%5Cgamma%20%5Calpha%7D%7B(1%2B%5Calpha%5E2)%5Cmu_i%7D%20%5Cvec%7Bn%7D_i%20%5Ctimes%20(%5Cvec%7Bn%7D_i%20%5Ctimes%20%5Cvec%7BB%7D%5E%5Cmathrm%7Beff%7D_i)%20%5C%5C%0A%20%20%20%20%20%20%20%20%26-%20%5Cdfrac%7B%5Calpha-%5Cbeta%7D%7B(1%2B%5Calpha%5E2)%7D%20u%20%5Cvec%7Bn%7D_i%20%5Ctimes%20(%5Chat%7Bj%7D_e%20%5Ccdot%20%5Cnabla_%7B%5Cvec%7Br%7D%7D%20)%5Cvec%7Bn%7D_i%0A%20%20%20%20%20%20%20%20%2B%20%5Cdfrac%7B1%2B%5Cbeta%20%5Calpha%7D%7B(1%2B%5Calpha%5E2)%7D%20u%20%5Cvec%7Bn%7D_i%20%5Ctimes%20(%5Chat%7Bn%7D_i%20%5Ctimes%20(%5Chat%7Bj%7D_e%20%5Ccdot%20%5Cnabla_%7B%5Cvec%7Br%7D%7D%20)%5Cvec%7Bn%7D_i)%0A%5Cend%7Balignedat%7D) and checking the contents in the browsers "inspect" feature.

So it seems to me that the `prefers-color-scheme: dark` has the unintended effect that if the browser is set to dark mode, the equations will be white and therefore vanish on the white background because RTD is not. I was able to verify the equations show up if I set my browser settings to "light theme".

**Comment by Sid8899** [comment]
Yes the problem was comming from there thanks, so to be able to have equations in Readthedocs website I changed my Edge navigator settings to light theme. And to be able to have them in markdown files(opened with VSCode) after spirit build, I had to change to light theme all Windows (W11). If I download a pdf of the docs from readthedocs website, equation still not apppear..

**Comment by GPMueller** [comment]
I don't know about the generated pdf, sorry. But at some point I believe the docs-page should be replaced by a docusaurus-generated one, which has its own light-/dark-mode switch, so I think the equations should always show up there. But I cannot tell when I'll have time to do that.
